packages/main/src/components/Form/Form.stories.mdx

```diff
@@ -52,9 +55,6 @@ import { DocsHeader } from '@shared/stories/DocsHeader';
             <CheckBox checked />
           </FormItem>
         </FormGroup>
-        <FormItem label={'Sole Form Item'}>
```

Comment:
The place of the sole form item was intended to be in the Middle of the form. Any specific reason to move it into the first place?

Reply:
I found it more explanatory to move it to the front. If the `Form` is displayed as one column, the sole form item can hardly be distinguished from the items inside a `FormGroup`:
<img width="500" alt="Screenshot 2020-11-11 at 08 54 21" src="https://user-images.githubusercontent.com/9749730/98784708-ce4f5080-23fb-11eb-9930-b1507fc5b3a1.png">

Sole item in front:
<img width="500" alt="Screenshot 2020-11-11 at 08 53 02" src="https://user-images.githubusercontent.com/9749730/98785051-4f0e4c80-23fc-11eb-948a-f7a1bac9a255.png">